tools/pio/pre_custom_esp82xx.py

```diff
@@ -63,6 +63,8 @@
     "-DUSE_TRIGONOMETRIC_FUNCTIONS_RULES",
     "-DUSE_CUSTOM_PROVISIONING",
 
+    "-DFEATURE_ESP_P2P", #for backwards compatibility
```

Comment:
I don't think this comment makes sense here, as it suggests the p2p feature is somehow deprecated.

Reply:
Suggested an alternative approach here: https://github.com/letscontrolit/ESPEasy/pull/4137#discussion_r922880583